docs/04-api-reference/front-commerce-core/config.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: "config"
+sidebar_position: 6
+description: "Config related methods"
+---
+
+<p>{frontMatter.description}</p>
+
+## `defineUserConfig`
+
+The defineConfig method allows to configure the Front-Commerce application.
+
+The method takes a [`UserConfig`](#userconfig) object as parameter. It can
+either be an object, a function that return an UserConfig or a Promise that
+resolve an UserConfig.
+
+```ts title="front-commerce.config.ts"
+import { defineConfig } from "@front-commerce/core/config";
+import themeChocolatine from "@front-commerce/theme-chocolatine";
+import magento2 from "@front-commerce/magento2";
+import storesConfig from "./app/config/stores";
+import cacheConfig from "./app/config/caching";
+
+export default defineConfig({
+  extensions: [magento2(storesConfig), themeChocolatine()],
+  stores: storesConfig,
+  cache: cacheConfig,
+  configuration: {
+    providers: [],
+  },
+  v2_compat: {
+    useApolloClientQueries: true,
+  },
+});
+```
+
+## `createConfigFromRequest`
+
+Create a configuration from request.
+
+:::warning
+
+This is an internal API, we recommend you to use the
+[`Front-Commerce context`](#) to retrieve configuration information
+
+:::
+
+## `getCurrentShopConfig`
+
+Get the current shop configuration.
+
+_Example_
+
+```ts
+import { getCurrentShopConfig } from "@front-commerce/core/config";
+
+const getAcmeConfig(config) {
+  return getCurrentShopConfig(config);
+}
+
+```
+
+## UserConfig
+
+### `extensions`
+
+_Optionnal_ Array of extensions to load
+
+### `stores`
+
+**Required** [Store configuration](#)
+
+### `cache`
+
+**Required** Your [cache configuration](#)
```

Comment:
issue: dead link

suggestion: remove link until it's documented. In the meantime, you can add an example here (using an import).

If no issue exists: create one in Linear and add this as a reminder

Reply:
fixed with your suggestion and created FC-1751 for follow-up